app/static/sass/styles.scss

```diff
@@ -0,0 +1,7 @@
+// This is just a sample SASS file.
```

Comment:
The [libsass-python Flask integration docs](http://hongminhee.org/libsass-python/frameworks/flask.html) suggest putting all SASS in `static/sass`, which is what we're doing here, but I'm not sure how I actually feel about it, since I think of the `static` directory as stuff that the user's browser will ultimately request as-is, which isn't the case with SASS. I guess it's not really a big deal either way, though, and it's not hard to alter if we change our minds.